### PR TITLE
Quoll v0.0.4

### PR DESCRIPTION
**Pull request by elainehoml**
Update to use Quoll v0.0.4

- Quoll now runs on the image data rather than image filenames
- Removes unnecessary dependencies from earlier versions of Quoll
- Adds progress bar (shows in the terminal)
- Allows users to choose calibration function (at the moment only no calibration or the RFI calibration function.)

### CODE REVIEW

File: src/okapi_em/_widget.py [hunk 1]
```diff
@@ -230,15 +231,15 @@ def ui_do_tab_ResMeas(self):
             vBox4.addLayout(qgrid3_layout)
             qgrid3_layout.addWidget(QLabel("tile size (px):"),0,0)
             self.spbQuolTileSize_px = QSpinBox()
-            self.spbQuolTileSize_px.setMinimum(8)
+            self.spbQuolTileSize_px.setMinimum(128)
```
Comment (perdigao1) [inline src/okapi_em/_widget.py:234]: why minimum of 128? Can you not run with 64 or 32?

File: src/okapi_em/_widget.py [hunk 2]
```diff
@@ -230,15 +231,15 @@ def ui_do_tab_ResMeas(self):
...
             self.spbQuolTileSize_px.setMaximum(10000)
             self.spbQuolTileSize_px.setValue(256)
             qgrid3_layout.addWidget(self.spbQuolTileSize_px, 0,1)
             
             #add pixel size box
             qgrid3_layout.addWidget(QLabel("pixel size (nm):"),1,0)
             self.spbQuolPixelSize_nm = QDoubleSpinBox()
-            self.spbQuolPixelSize_nm.setMinimum(0.001)
+            self.spbQuolPixelSize_nm.setMinimum(1e-6)
```
Comment (perdigao1) [inline src/okapi_em/_widget.py:242]: not sure why this change, but ok. I don't think any microscope nowadays can do femtometer resolution

File: src/okapi_em/_widget.py (hunk 1 above)
Comment (elainehoml) [reply src/okapi_em/_widget.py:234]: No, Miplib doesn't allow anything below 128

File: src/okapi_em/_widget.py (hunk 2 above)
Comment (elainehoml) [reply src/okapi_em/_widget.py:242]: Was a just in case, so if we decide to make the pixel size field more general and allow users to specify units then this would at least make sure it's not zero
